src/manifest.c

```diff
@@ -688,7 +688,8 @@ struct manifest *load_manifest(int current, int version, struct file *file, stru
 {
 	struct manifest *manifest = NULL;
 	int ret = 0;
-	bool retried = false;
+	int retried = 0;
```

Comment:
Artifact of debugging, retested with sane code and things are all in order still.